integration_tests/integration_tests.feature

```diff
@@ -129,14 +129,14 @@ Feature: Run integration tests
      |  default_3a  |
      |  default_3b  |
 
-  Scenario: Cumulative Grid Trades is the same with and without keeping the past markets
-     Given we have a scenario named default_2a
+  Scenario: Cumulative Grid Trades are the same with and without keeping the past markets
+     Given we have a scenario named balancing_market.default_2a
```

Comment:
Why changing to the balancing_market.default_2a ?

Reply:
I want to test if cumulative_grid_balancing_trades too are indifferent of using past OR current market for calculating results